### PR TITLE
reduce axioms in number elementhood proofs

### PR DESCRIPTION
**Pull request by icecream17**
rename 2cnOLD -> 2cnALT, as noted in https://github.com/metamath/set.mm/pull/2872
and also reorder rpssre

### CODE REVIEW

**Comment by benjub** [comment]
To me, mergeable after that typo is fixed.

**Review by wlammen — CHANGES_REQUESTED** [review]
I think you should add rpreOLD and rpssreOLD.  Otherwise well done. Reduction in axiom usage confirmed for all but rpre and rpssre.  The latter are an obvious shortening.

**Comment by benjub** [comment]
> I think you should add rpreOLD and rpssreOLD. Otherwise well done. Reduction in axiom usage confirmed for all but rpre and rpssre. The latter are an obvious shortening.

To me, the swap rpre <-> rpssre is straightforward enough that there is no need to keep OLD versions, but if @wlammen thinks there is, then let it be so.

File: changes-set.txt [hunk 1]
```diff
@@ -27,6 +27,7 @@ make a github issue.)
 
 DONE:
 Date      Old       New         Notes
+ 8-Oct-22 2cnOLD    2cnALT
```
Comment (benjub) [inline changes-set.txt:30]: Not needed since we have 2cn with identical statement anyway.

**Comment by wlammen** [comment]
> if @wlammen thinks there is, then let it be so.

Why only do I feel being criticized here?

I don't know why my personal opinion plays any role here.  I almost never look at OLD versions, and I am often annoyed because they litter pages like 'Nearby theorems'.  That's why I am often keen to remove outdated ones early.

But it is an established habit to create these, for whatever reason, and there might be people around regarding them as important.  
So I let general ideas prevail.

**Comment by avekens** [comment]
I also think rpreOLD and rpssreOLD should be added. Afterwards, i will accept this PR.

**Comment by benjub** [comment]
> Why only do I feel being criticized here?

That was really not my intent. Quite the opposite: it's because I know you have experience with this that I say "if @wlammen  thinks this, then let it be so". If that came from someone totally new to set.mm, I would be more likely to go over their opinion.

**Comment by icecream17** [comment]
this took a while, sorry


I made a new pr with all the commented updates:
https://github.com/metamath/set.mm/pull/2888

**Pull request closed without merging**